pages/docs/projects/tangle-network/overview.mdx

```diff
@@ -5,24 +5,63 @@ description: General overview of the Tangle Network.
 
 import { UseCasesArea, ParticipateArea, TestNetworkArea } from "../../../../components/UseCasesTangle";
 import FullWebbCTA from "../../../../components/FullWebbCTA";
+import SvgComponent from "../../../../components/TangleOverviewSVG.tsx";
 
 # Tangle Network Overview
 
 The Tangle Network is a specialized platform engineered to support and govern cross-chain Zero-Knowledge (ZK) applications. Leveraging the robust Substrate blockchain framework, the Tangle Network forms the core infrastructure to facilitate a new generation of privacy-enhancing and governance-decentralized applications.
 
-The Tangle Network's unique offering centers around the intersection of cross-chain functionality, advanced governance systems, and the world of ZK applications. Our network seamlessly merges these distinct elements, creating an ecosystem that amplifies the strengths of each component.
+The Tangle Network's unique offering centers around the intersection of cross-chain functionality, familiar EVM tooling and compatibility, advanced governance systems, and the world of ZK applications. Our network seamlessly merges these distinct elements, creating an ecosystem that amplifies the strengths of each component.
 
-### Powering Cross-chain ZK Applications
+## Key Features
 
-The primary objective of the Tangle Network is to cater to cross-chain ZK applications, offering a tailored environment for their operation and governance. Our network transcends traditional chain boundaries, providing a highway for these advanced applications to interact, exchange data, and perform seamlessly across different blockchain networks.
+### Cross-Chain Functionality and EVM Compatibility
 
-### Built on [Substrate](https://substrate.io/)
+The Tangle Network breaks traditional chain boundaries, allowing seamless interaction, data exchange, and operation of ZK applications across different blockchain networks. Leveraging the power of the Ethereum Virtual Machine (EVM) on Substrate, developers can utilize familiar EVM tooling and compatibility to build decentralized applications (DApps), create Non-Fungible Tokens (NFTs), and utilize ERC20 tokens across multiple networks.
 
-The Tangle Network is built on the Substrate, an advanced blockchain framework that grants us flexibility, robustness, and cutting-edge features. This strategic choice ensures that our platform stays ahead in terms of speed, security, and scalability, proving to be a reliable backbone for our network.
+### Advanced Governance
 
-### With Unprecedented [Governance](tangle-governance.mdx)
+The Tangle Network implements an innovative governance model based on Distributed Key Generation (DKG) protocol. The DKG protocol serves as a security instrument for the Tangle Network's Anchor System, ensuring the validity of bridge updates through DKG-signed signatures.
```

Comment:
What are your thoughts on conveying this as well?
1. Applications can _propose_ to be governed by Tangle using Substrate Democracy
2. The tangle token holders can approve/reject this **genesis proposal**.
3. If approved, Tangle will sign its payloads, if rejected it wouldn't.
```suggestion
The Tangle Network implements an innovative governance model based on Distributed Key Generation (DKG) protocol. The DKG protocol serves as a security instrument for the Tangle Network's cross-chain zkApps, ensuring the validity of bridge updates through Tangle's threshold-signatures.
```